api-client/src/test/java/com/xing/api/JsonSerializationTest.java

```diff
@@ -304,6 +314,44 @@ private static void assertPhotoUrls(ProfileVisit visit) {
         assertThat(visit.photoUrls()).isEqualTo(urls);
     }
 
+    private static void assertPrimaryInstitutionName(XingUser user) {
```

Comment:
I think these tests should live in `com.xing.api.data.profile.XingUserTest` class, since they have nothing to do with the serialization of the object.